src/components/Questionnaire/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+
+class Questionnaire extends Component {
+
+  render() {
+    const {sampleQuestionnaire} = this.props;
+    return (
+        <div className="questionnaire__wrapper">
+          <h2 className="questionnaire__item--title">Questionnaire</h2>
+          {sampleQuestionnaire.map((item, index)=>{
+          return (
+            <div key={index} className="questionnaire__item">
+
+                <p className="questionnaire__item--question">
+                    {item.question}
+                </p>
+                <p className="questionnaire__item--answer">
+                    {item.answer}
+                </p>
+            </div>
+          );
+          })}
+        </div>
+    );
+  }
+}
+Questionnaire.propTypes = {
+  sampleQuestionnaire: PropTypes.array
```

Comment:
I think we should write `.isRequired` here, because even though the component is not required, if the user does add some info here, the component should get the sampleQuestionnaire prop from its parent component!

Reply:
that's right, good point!